src/app/car.ts

```diff
@@ -0,0 +1,10 @@
+export class Car {
+
+    id: string
```

Comment:
id won't always be informed, so should be optional (id?: string;)

Reply:
Fixed.